### ISSUE TITLE
Ruleset cis-02-29-2016.json contains malformed JSON

### ISSUE THREAD

**Issue by brunns** [nccgroup/Scout2#217]
As of revision 1ad27b8, cis-02-29-2016.json is reported as malformed.

    git co 3f8e672
    python setup.py install && Scout2 --ruleset cis-02-29-2016.json --no-browser

works fine.

    git co 1ad27b8
    python setup.py install && Scout2 --ruleset cis-02-29-2016.json --no-browser

fails with:

    Error: ruleset filecis-02-29-2016.json contains malformed JSON

**Comment by l01cd3v** [comment]
Thanks ! Should be fixed now - Note that this ruleset is not a complete match w/ the CIS benchmark  as Scout2 does not support several services related to monitoring

**Comment by brunns** [comment]
Cool, thank you very much! We may be able to commit some time to add any services which aren't currently supported - do you have a list of them anywhere?

**Comment by l01cd3v** [comment]
I don't have the list handy, but it's pretty much the difference between services mentioned in the CIS benchmark and the list of services you see in the Scout2 navbar.

I have a number of higher priority issues to take care of so adding support for these may not happen for a while; PRs are welcome though :) Check out the [wiki](https://github.com/nccgroup/Scout2/wiki) if you're curious about the tools internals and how to fetch config for new services and create rules.

Basically, the main reason for not putting much effort towards implementing checks for the logging/monitoring section is that most AWS accounts I've reviewed had logging/monitoring implemented outside of the AWS account.
